ApplicationLibCode/ReservoirDataModel/RigCaseCellResultsData.cpp

```diff
@@ -1987,6 +2000,112 @@ void RigCaseCellResultsData::computeDepthRelatedResults()
     }
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RigCaseCellResultsData::computeIndexResults()
```

Comment:
@magnesj Computes all three directions when the first is requested. Can break it into three to save memory maybe?

Reply:
Compute all when one is required is OK